src/Locations.py

```diff
@@ -30,7 +34,16 @@ class Locations(object):
     '''
 
     def __init__(self, starting_url):
-        self.starting_url = starting_url
-        page = urlopen(starting_url)
-        dataset_download_html = page.read()
-        dataset_start_html = dataset_download_html.decode("utf-8")
+        self.url = starting_url
+        page = requests.get(self.url)
+        soup = BeautifulSoup(page.content, "html.parser")
```

Comment:
this is going to be replaced by the file: src/rrc_scrape.py

https://github.com/mlbelobraydi/TXRRC_data_harvest/pull/36/commits/98ef9660c5db4d94db4b83097ac750d21012203d#diff-c363c527d148d304224201f4fca2365f786209b56d18348a29db7cc9babec445